### PR TITLE
[Doc] Added description of the tag_with_exception_message option

### PR DESCRIPTION
**Pull request by andsel**
Update the documentation to describe the `tag_with_exception_message` option

### CODE REVIEW

**Review by karenzone — CHANGES_REQUESTED** [review]
This will break the docs build.  I will track down the problem and come back with a suggested fix.  In the meantime, **do not merge!**

File: docs/index.asciidoc [hunk 1]
```diff
@@ -222,5 +223,15 @@ of your ruby script file defined in `path`.
 Tag to add to events in case the ruby code (either inline or file based)
 causes an exception.
 
+[id="plugins-{type}s-{plugin}-tag_with_exception_message"]
+===== `tag_with_exception_message`
+
+* Value type is <<boolean,boolean>>
+* Default value is `false`
+
+If `true` adds a tag to the event that's the concatenation of <<plugins-{type}s-{plugin}-common-tag_with_exception_message>>
```
Comment (karenzone) [inline docs/index.asciidoc:232]: Here's the problem... There is no anchor for <<plugins-{type}s-{plugin}-common-tag_with_exception_message>> to resolve to.

**Review by karenzone — CHANGES_REQUESTED** [review]
Comment inline.

File: docs/index.asciidoc (hunk 1 above)
Comment (andsel) [reply docs/index.asciidoc]: By bad, an error in cut and paste

File: docs/index.asciidoc [hunk 2]
```diff
@@ -222,5 +223,15 @@ of your ruby script file defined in `path`.
 Tag to add to events in case the ruby code (either inline or file based)
 causes an exception.
 
+[id="plugins-{type}s-{plugin}-tag_with_exception_message"]
+===== `tag_with_exception_message`
+
+* Value type is <<boolean,boolean>>
+* Default value is `false`
+
+If `true` adds a tag to the event that's the concatenation of <<plugins-{type}s-{plugin}-tag_with_exception_message>>
```
Comment (karenzone) [inline docs/index.asciidoc:232]:
```suggestion
If `true` adds a tag to the event that is the concatenation of `tag_with_exception_message`
```

Comment (karenzone) [reply docs/index.asciidoc:232]: Let's hard code this. A link inside the section it's linking to doesn't make sense. 

Screen Shot 2021-06-01 at 7 10 59 PM

**Review by karenzone — APPROVED** [review]
Left a suggestion inline for consideration.  Otherwise, builds cleanly and LGTM

**Pull request merged**